praw/reddit.py

```diff
@@ -542,6 +542,44 @@ def generator(url):
 
         return generator(url)
 
+    def objectify_request(
```

Comment:
While we may not end up needing this, let's try to avoid introducing new "public" methods.

Reply:
I noticed that all of the public methods, such as get, patch, is just the same code but with one changed string. I could make this a private method if needed.

Reply:
That would be better if the method is needed.